data/sea/54-zebra.h

```diff
@@ -3,6 +3,9 @@
 #if !ICICLE_NO_INPUT
 #if ICICLE_ZEBRA
 
+/* Number of facts to read before starting compute and checking allocation limit */
+static const iint_t zebra_chunk_fact_count = 2;
```

Comment:
is it worth putting this in `zebra_config_t` to make it easier to quickcheck with different limits?